l3build/l3build.lua

```diff
@@ -1643,8 +1629,16 @@ function cmdcheck()
   for _,i in ipairs(typesetsuppfiles) do
     cp(i, supportdir, testdir)
   end
+<<<<<<< HEAD
```

Comment:
That was a rebase gone wrong. It is now fixed and properly rebased on the current master.